scripts/update-rules.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// Automatically regenerates the rules table in readme.md.
+
+const fs = require('fs');
+const path = require('path');
+const {rules, configs} = require('../');
+
+const pathReadme = path.resolve(__dirname, '../README.md');
+const readmeContents = fs.readFileSync(pathReadme, 'utf8');
+const tablePlaceholder = /<!--RULES_TABLE_START-->[\S\s]*<!--RULES_TABLE_END-->/;
+
+// Config/preset/fixable emojis.
+const EMOJI_RECOMMENDED = ':white_check_mark:';
+const EMOJI_FIXABLE = ':wrench:';
+
+// Generate rule table contents.
+const rulesTableContent = Object.keys(rules).filter(ruleName => !rules[ruleName].meta.deprecated)
+	.sort()
+	.map(ruleName => {
+		// Check which emojis to show for this rule.
+		const isRecommended = Object.prototype.hasOwnProperty.call(configs.recommended.rules, `unicorn/${ruleName}`);
```

Comment:
This check is faulty. It should not mark the rule as recommended if it's in the recommended preset as off.

Reply:
Good catch, fixed.